Client/Ecosia/Experiments/Unleash/APNConsentOnLaunchExperiment.swift

```diff
@@ -0,0 +1,30 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Core
+
+struct APNConsentOnLaunchExperiment {
+    private init() {}
+    
+    static var toggleName: Unleash.Toggle.Name {
+        .apnConsentOnLaunch
+    }
+
+    static var isEnabled: Bool {
```

Comment:
I believe this can be `private` as not needed elsewhere (same for the `toggleName`). 
It might also be prone to ad-hoc business logic while what we end up using is the  `requestAPNConsentIfNeeded(:)` really.